src/chainstate/stacks/db/unconfirmed.rs

```diff
@@ -755,13 +825,9 @@ mod test {
                     let signed_mempool_txs = signed_txs
                         .into_iter()
                         .map(|tx| {
-                            MemPoolTxInfo::from_tx(
-                                tx,
-                                0,
-                                consensus_hash.clone(),
-                                stacks_block.block_hash(),
-                                tenure_id as u64,
-                            )
+                            let mut bytes = vec![];
+                            tx.consensus_serialize(&mut bytes).unwrap();
```

Comment:
The `StacksMessageCodec` trait provides a handy `serialize_to_vec` method as a convenience to avoid needing to write this boiler plate.